commands/emoji.js

```diff
@@ -14,6 +18,14 @@ const Module = new Augur.Module()
     let id = test.exec(suffix);
     let ext = (id[1] ? ".gif" : ".png");
     if (id) msg.channel.send({files: [{name: id[2] + ext, attachment: `https://cdn.discordapp.com/emojis/${id[3]}.${(id[1] ? "gif" : "png")}`}]});
+    else if (emojiUnicode(suffix)){
+      request(`https://twemoji.maxcdn.com/v/latest/svg/${unicode(args)}.svg`, async function(err, response, body){
```

Comment:
Neither `unicode` nor `args` are defined. Is that meant to be `emojiUnicode(suffix)`?

I presume `emojiUnicode(suffix)` has been tested, as it shouldn't be passing the actual unicode character in the suffix. (It'd be `:bed:` or similar.) Per the docs, it *appears* that this needs yet another lib, `emoji-name-map`.

Reply:
Yeah, another case of forgetting to change my var names before implementing. It doesn't look like that library is required though.